### PR TITLE
Crude CircuitBoards

### PR DESCRIPTION
**Pull request by HourlyLimit**
## About The Pull Request

**_I will make sprites later for this for now it will be placeholder Circuit Boards and other placeholders also buffed potato cell_**
I have made **Crude Circuit Boards** so people could make ships from nothing more than a pickaxe here is a list of what I added.

- Added **Crude Engine**
- Added **Crude Cave Pacman** _(Cave name by Bread Man)_
- Added **Crude Helm Board**
- Added **Crude Board Component**
- Added **Crude Board**
- Added **Craftable Coil**
- Added **Craftable Blueprints**
- Added **Crude APC board**
- Added **Crude Autolathe Board**
- Added **Craftable Potato Cell**
- Added **Craftable Screwdriver**
- Added **Craftable Crowbar**
- Added **Craftable Wrench**
- Added **Craftable Capacitor**
- Added **Craftable Iron for a big waste of raw iron**

## Why It's Good For The Game

Makes Blackpill Possible to get RnD from and people can land their ship and run off on their own and make their own ship from nothing i think that would be a fun idea for people to do.

## Changelog
:cl:
add: Added Crude Engine
add: Added Crude Cave Pacman
add: Added Crude Helm Board
add: Added Crude Board Component
add: Added Crude Board
add: Added Craftable Coil
add: Added Craftable Blueprints
add: Crude APC board
add: Crude Autolathe Board
add: Craftable Potato Cell
add: Added Screwdriver
add: Craftable Crowbar
add: Craftable Wrench
add: Craftable Iron for a big waste of raw iron
add: Craftable Capacitor
/:cl:
image

### CODE REVIEW

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -347,8 +347,8 @@
 	desc = "A rechargeable starch based power cell."
 	icon = 'icons/obj/hydroponics/harvest.dmi'
 	icon_state = "potato"
-	charge = 100
-	maxcharge = 300
```
Comment (SuperNovaa41) [inline code/modules/power/cell.dm:351]: why the subtle potato cell buff

Comment (HourlyLimit) [reply code/modules/power/cell.dm:351]: its debuffed it should be

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 2]
```diff
@@ -221,6 +236,14 @@
 		/obj/item/stack/cable_coil = 2,
 		/obj/item/stock_parts/subspace/filter = 1)
 
+/obj/item/circuitboard/machine/pacman/crude
+	name = "CAVEMAN-type Generator (Machine Board)"
+	icon_state = "engineering"
+	build_path = /obj/machinery/power/port_gen/pacman/crude
+	req_components = list(/obj/item/stack/ore/iron = 2,
```
Comment (GuiltyNeko) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm:244]: You'll need a capacitor in here or else you'll get zero power output in port_gen.dm when it grabs the temp rating.

**Review by GuiltyNeko — CHANGES_REQUESTED** [review]
To be clearer, and to request changes officially, the pacman needs a capacitor or it won't work, and will runtime. This needs to be fixed before this can be merged.

**Comment by Addust** [comment]
grug boardman

File: code/datums/components/crafting/recipes.dm [hunk 3]
```diff
@@ -404,6 +404,14 @@
 	time = 25
 	category = CAT_MISC
 
+/datum/crafting_recipe/helm
```
Comment (GuiltyNeko) [inline code/datums/components/crafting/recipes.dm]: This has the same datum name as the recipe below it, so it gets overwritten by that one, making this recipe uncraftable.

**Review by GuiltyNeko — CHANGES_REQUESTED** [review]

**Comment by GuiltyNeko** [comment]
Test Merging

**Review by GuiltyNeko — CHANGES_REQUESTED** [review]
And the autolathe needs a matter bin to hold materials.

**Pull request closed without merging**